dobby_security.sh

```diff
@@ -24,40 +24,36 @@ crunVersion=$(crun --version)
 
 
 usage () {
-  cat <<EOF
-
-Checks for dozens of common best-practices around deploying Dobby containers in production.
-Based on the CIS Docker Benchmark 1.3.1.
-Usage: ./dobby_security.sh -c Netflix [OPTIONS] 
-Options:
-  -c    mandatory  Container name (Ensure the container is running)
-  -h    optional   Print this help message
-  -v	  optional   Print debug messages
-  -t    optional   Comma delimited list of specific test(s) id
-  -e    optional   Comma delimited list of specific test(s) id to exclude
-  -b    optional   Don't use colors (currently not supported)
-
+cat <<EOF
```

Comment:
Should be indented 4 spaces.

Reply:
There was misunderstanding, the line:
`cat <<EOF`
Needs to be indented 4 spaces (as it was previously 2 spaces), all other should not as it would make them indented in the output.